worker-message-prioritization-rabbitmq/src/main/java/com/github/workerframework/workermessageprioritization/rabbitmq/RabbitManagementApi.java

```diff
@@ -41,8 +41,8 @@
 
 public class RabbitManagementApi <T> {
 
-    private static final int READ_TIMEOUT_SECONDS = 10;
-    private static final int CONNECT_TIMEOUT_SECONDS = 10;
+    private static final int READ_TIMEOUT_SECONDS = 20;
+    private static final int CONNECT_TIMEOUT_SECONDS = 20;
```

Comment:
Increasing these a bit as there is a few socket exceptions that occur after 10 seconds in the log attached to https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=691034